src/Tracing/Tracer.php

```diff
@@ -58,7 +58,7 @@ public function setActiveSpan(Span $span): Span
     public function createSpan(string $name): Span
     {
         $parent = $this->getActiveSpan()->getContext();
-        $context = SpanContext::fork($parent->getTraceId());
+        $context = new SpanContext($parent->getTraceId(), $parent->getSpanId());
```

Comment:
I think this is the way we want to handle new SpanContext generation, but I think we should discuss this.